roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml

```diff
@@ -49,12 +49,28 @@
     state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
   when: client_group_name in group_names
 
-- name: install red hat storage nfs gateway
+- name: install red hat storage NFS gateway
   apt:
-    name: nfs-ganesha-fsal
+    name: nfs-ganesha
     state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
   when: nfs_group_name in group_names
 
+- name: install red hat storage nfs file gateway
```

Comment:
@leseb  @ktdreyer I don't need to do any repo configuration for nfs-ganesha if someone is running rhcs as long as this is set to true (https://github.com/ceph/ceph-ansible/blob/wip-rgw-nfs/group_vars/all.yml.sample#L161) right?

If someone is running rhcs, nfs-ganesha and it's fsal and ntirpc repos will be in the same repo as the rest of the ceph bits come from right?

Reply:
Right